nion/swift/ExportDialog.py

```diff
@@ -33,7 +34,8 @@
     from nion.swift.model import DisplayItem
 
 _ = gettext.gettext
-
+CM_PER_INCH = 2.54
+PIXELS_PER_INCH = 96
 
```

Comment:
There should be exactly two blank lines before each top-level class. Please add one here.